_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,283 @@
+```
+
+#### Passed example 4
+
+Multiple types of required owned elements are present.
+
+```html
+<ul role='menu'>
+	<li></li> <!-- implicit role -->
+	<li role='menuitem'></li>
+	<li role='menuitemradio'></li>
+</ul>
+```
+
+#### Passed example 5
+
+Required owned element does not have to be an immediate child of the role, but could be any descendant. (See accessibility support notes for details).
+
+```html
+<div role="table">
+	<div>
+		<div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 6
+
+Nested required owned elements.
+
+```html
+<div role="grid">
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 7
+
+Element with role `list` contains a required owned element which has an implicit role of `listitem`.
+
+```html
+<div role='list'>
+   <li>
+   </li>
+</div>
+```
+
+#### Passed example 8
+
+Element with role `menubar` can have multiple required owned element (`menuitem` `menuitemcheckbox` and `menuitmeradio`) but only one is necessary.
+
+```html
+<div role='menubar'>
+   <div role='menuitem'>
+   </div>
+   <div role='menuitemcheckbox'>
+   </div>
+	 <div role='menuitmeradio'>
+   </div>
+</div>
+```
+
+#### Passed example 9
+
+Element with role `list` contains a required `listitem`, the relationship between both elements is made possible through the `aria-owns` properties.
+
+**Note:** This test case follows the definition of [owned by](#owned-by) used in this rule. If implemented differently, this definition could cause differences in outcome for this test case.
+
+```html
+<div role="list" aria-owns="id1">
+</div>
+
+<div id="id1" role="listitem"></div>
+```
+
+#### Passed example 10
+
+Element with role `grid` has at least one required owned element with role `rowgroup`.
+
+```html
+<div role="grid">
+	<div>
+		<div role="row">
+			<span>Item 1</span>
+		</div>
+	<div>
+	<div role="rowgroup">
+		<div role="row">
+			<span>Item 2</span>
+		</div>
+	</div>
+</div>
+```
+
+### Failed
+
+#### Failed example 1
+
+Element with role `list` is missing required owned element `listitem`.
+
+```html
+<ul role='list'>
+	<span>
+	</span>
+</ul>
+```
+
+#### Failed example 2
+
+Element with role `tablist` is missing required owned element `tab`.
+
+```html
+<ol role='tablist'>
+	<li role='listitem'>
+	</li>
+</ol>
+```
+
+#### Failed example 3
+
+Element with role `grid` is missing a required owned element `rowgroup`, even though the containing required owned element `row` is present.
```

Comment:
The required owned elements for `grid` are `row` and `rowgroup`. Since the expectation requires just one instance of the required owned elements, this is a pass.

Reply:
Nice catch, I have updated this to something that will fail.